app/src/main/java/zegreanu/cristi/geoquiz/QuizActivity.kt

```diff
@@ -24,15 +32,16 @@ class QuizActivity : AppCompatActivity() {
         Question(R.string.question_americas, true),
         Question(R.string.question_asia, true)
     )
+
+    private var isCheater by Delegates.notNull<Boolean>()
```

Comment:
I think by default this can be false - since it's fair to say that if the user didn't interact with CheatActivity (including not pressing showAnswerButton) this means that he's not a cheater.

Reply:
Yes, otherwise there is a bug with ch5 challenges anyway